test/net/sourceforge/kolmafia/persistence/ItemFinderTest.java

```diff
@@ -900,4 +903,35 @@ public void itShouldParseMultipleItems() {
     assertEquals(results[0].getCount(), 1);
     StaticEntity.setContinuationState(MafiaState.CONTINUE);
   }
+
+  @Test
+  public void itShouldFindSomeEdgeCasesSatisfactorily() {
```

Comment:
Can you split this into multiple tests?

Reply:
Sure.  Since I could not completely figure out the logic for grouping the existing tests together I just bundled mine all together :-)

Reply:
Better?  You may resolve if so.